# Add from CAD file
parent_frame: "map"
child_frame: "vins_odom"
x: 1.0
y: 0.0
z: 0.0
roll: 0.0
pitch: 0.0
yaw: 0.0
